### ISSUE TITLE
forcetk.Client.prototype.blob code has broken variable name

### ISSUE THREAD

**Issue by elijahdorman** [developerforce/Force.com-JavaScript-REST-Toolkit#74]
The signature for the function is
```javascript
    forcetk.Client.prototype.blob = function(path, fields, filename, payloadField, payload, callback, error, retry) { 
```
But when the blob is called, the variable name is changed

```javascript
                    } else if(request.status == 401 && !retry) {
                        that.refreshAccessToken(function(oauthResponse) {
                            that.setSessionToken(oauthResponse.access_token, null,oauthResponse.instance_url);
                            that.blob(path, fields, fileName, file, callback, error, true);
                        },
                        error);
                    } else {
```

A quick search of the file shows that `fileName` does not occur anywhere else in the file (and my linter agrees).

**Comment by metadaddy-sfdc** [comment]
Wow - how did that happen? Thanks for filing this, @elijahdorman!

**Comment by elijahdorman** [comment]
@metadaddy-sfdc I didn't mention in my previous comment (my mistake) that the same line, the variable `file` is also an undefined variable (that name also doesn't occur anywhere else in the file).

**Comment by metadaddy-sfdc** [comment]
Fixed in e12fc02
